app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -411,7 +411,7 @@ private Account getAccountWithEmail(Study study, String email) {
     }
     
     private void rethrowResourceException(ResourceException e, Account account) {
-        logger.info(String.format("Stormpath error: %s: %s", e.getCode(), e.getMessage()));
+        //logger.info(String.format("Stormpath error: %s: %s", e.getCode(), e.getMessage()));
```

Comment:
Added back (seems useful to have in the logs, didn't mean to leave it commented out)